src/actuator/ActuatorFunctorsFAST.C

```diff
@@ -0,0 +1,57 @@
+// Copyright 2017 National Technology & Engineering Solutions of Sandia, LLC
+// (NTESS), National Renewable Energy Laboratory, University of Texas Austin,
+// Northwest Research Associates. Under the terms of Contract DE-NA0003525
+// with NTESS, the U.S. Government retains certain rights in this software.
+//
+// This software is released under the BSD 3-clause license. See LICENSE file
+// for more details.
+//
+
+#include <actuator/ActuatorFunctorsFAST.h>
+#include <NaluEnv.h>
+
+namespace sierra
+{
+namespace nalu
+{
+
+template<>
+ActFastUpdatePoints::ActuatorFunctor(ActuatorBulkFAST& actBulk):
+  actBulk_(actBulk)
+{
+  TOUCH_DUAL_VIEW(actBulk_.pointCentroid_, memory_space);
```

Comment:
Will likely generate pedantic warnings of trailing semicolon.

Reply:
Okay. I will address this.